metrics-facade-prometheus/src/main/java/com/ringcentral/platform/metrics/samples/prometheus/PrometheusSampleMaker.java

```diff
@@ -13,22 +13,66 @@
 
 import java.util.List;
 
-import static io.prometheus.client.Collector.Type.GAUGE;
-import static java.util.Collections.*;
+import static io.prometheus.client.Collector.Type.*;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
 import static java.util.stream.Collectors.toList;
 
 public class PrometheusSampleMaker implements SampleMaker<
     PrometheusSample,
     PrometheusSampleSpec,
-    PrometheusInstanceSampleSpec> {
+    PrometheusInstanceSampleSpec,
+    PrometheusInstanceSample> {
+
+    public static boolean DEFAULT_SEPARATE_HISTOGRAM_AND_SUMMARY = true;
```

Comment:
should it be private and final?

Reply:
It should be final, of course. I'll fix it. 
It also should be public to be used in the builder class. Currently, it isn't used in the builder class and I'll fix it as well.
Thanks.

Reply:
Fixed.